cibuildwheel/macos.py

```diff
@@ -58,14 +58,11 @@ def shell(args, env=None, cwd=None):
             before_build_prepared = prepare_command(before_build, python=python, pip=pip)
             shell(shlex.split(before_build_prepared), env=env)
 
-        # install the package first to take care of dependencies
-        shell([pip, 'install', project_dir], env=env)
-
         # build the wheel to temp dir
         temp_wheel_dir = '/tmp/tmpwheel%s' % config.version
         shell([pip, 'wheel', project_dir, '-w', temp_wheel_dir, '--no-deps'], env=env)
-        temp_wheel = glob(temp_wheel_dir+'/*.whl')[0]
 
+        temp_wheel = glob(temp_wheel_dir+'/*.whl')[0]
```

Comment:
This is still an open question. I still don't know of any projects where multiple wheels get built so I decided not to make any changes, at least for this PR.